blog/2023-11-27-magazine-006.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Inflight Magazine no. 6
+description: Updates from the winglang project
+authors: 
+  - revitalb
+  - uric
+tags: [cloud-oriented programming, winglang, changelog, newsletter,]
+hide_table_of_contents: true
+---
+> The 6th issue of the Wing Inflight Magazine.
+> <!--truncate-->
+
+Hey folks!
+
+We’re back with another issue of the **Wing Inflight Magazine** and we are excited to share some updates about [winglang](https://winglang.io).
+
+- 🚀 [New: bring ui](#bring-ui)
+- 🚀 [New: Singleton](#singleton)
+- 🔥 [Remove $ from String Interpolations](#remove_dollar_from_string_interpolations)
+- 🚀 [New: Expose AWS Resources](#aws-resources)
+- 🚀 [New: Sourcemaps and Stacktraces](#sourcemaps-and-stacktraces)
+- 📅 [Upcoming Events](#upcoming-events)
+- 🎥 [The Wingly Update](#the-wingly-update)
+
+### bring ui
+
+Introducing the ui module to the SDK, a collection of APIs for adding visual interactions for your classes in the Wing Console. UI components aren't displayed in the Wing Console quite yet, but they can be accessed and called through the simulator API!
+
+![bring_us](./assets/2023-11-27-magazine-006/bring_ui.png)
+
+❤️ [Chris Rybicki](https://github.com/Chriscbr)❤️
+
+### Singleton
+
+If you want to create one instance of a resource thats the way to go!
+We are not making it easy to access the root app from *every* node through `Node.of(this).app` or `Node.of(this).root`. Both return the same root app, even if it's not the absolute root of the tree. `tryFindChild(id)` is also added to the app to make it easier to implement app-level singletons!
+
+A few use cases within Wing for a singleton include: When adding a provider and ensuring the singular presence of it. Another scenario involves implementing diverse checks for storing resources in a unified global bucket, with the aim of maintaining a singular presence.
+
+Check this out:
+![mutarray_1](./assets/2023-11-27-magazine-006/singleton.png)
+
+
+❤️ [Elad Ben-Israel](https://github.com/eladb) ❤️
+
+
+### Remove `$` from `String` Interpolations
+
+#### This one is a life changer :)
+Before this change, we used the `$` for String interpolations, but not anymore! 
+The motivation behind this one is *one less char to type*, more ergonomic and the `$` char had no practical use.  
+In our context, we're using an expression inside the `string` interpolation so we need a start and end token (the curly braces) but anything else seems redundant.
+
+This is a heck big of a deal! and it probably breaks a lot of Wing apps... but we believe its really worth it!
+
+![remove_dollar_from_string_interolations](./assets/2023-11-27-magazine-006)
```

Comment:
if this image contains code, please use a markdown codeblock instead (can use chatGPT)